### PR TITLE
i18n SC-5049 Translation Static Scripts dataprivacy

### PR DESCRIPTION
**Pull request by DevSchmidtchen**
Ticket: https://ticketsystem.schul-cloud.org/browse/SC-5049

### CODE REVIEW

**Comment by Parnswir** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5726273&bid=18734660)

**Comment by Parnswir** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5726273&bid=18734660)

**Comment by Parnswir** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5726273&bid=18734660)

**Comment by Parnswir** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5726273&bid=18734660)

**Comment by Parnswir** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5726273&bid=18734660)

File: static/scripts/dataprivacy/registration-link-validation.js [hunk 1]
```diff
@@ -1,26 +1,20 @@
 window.addEventListener('DOMContentLoaded', () => {
 	/*
-	// invalid registration detexted
+	// invalid registration detected
```
Comment (dominikbruechner) [inline static/scripts/dataprivacy/registration-link-validation.js:3]: 👏

**Comment by Parnswir** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5726273&bid=18734660)

**Comment by Parnswir** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5726273&bid=18734660)

**Pull request merged**